src/main/java/com/easypost/model/PaymentMethod.java

```diff
@@ -0,0 +1,106 @@
+package com.easypost.model;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+import com.easypost.net.EasyPostResource;
+
+public class PaymentMethod extends EasyPostResource {
+    private String id;
+    private String object;
+    private PrimaryPaymentMethod primaryPaymentMethod;
+    private SecondaryPaymentMethod secondaryPaymentMethod;
+
+    /**
```

Comment:
Functions need to come after the properties.

Reply:
I would have thought the linter would have caught this.

Reply:
As would I. These are actually out of order all throughout the lib. I've created an internal task to circle back to cleaning these up.